src/components/Calculator.js

```diff
@@ -1,11 +1,17 @@
+import PropTypes from 'prop-types';
+
+const Display = ({ output }) => (
+  <div className="display">{ output }</div>
+);
```

Comment:
- [ ] good job making the changes requested, however as mentioned by the previous reviewer https://github.com/lengushuru/math-magician/pull/2#discussion_r1143899728 , please notice that this should be a component itself and should be in a separate file just like Calculator.js, then you would import this component here using the import statement and use it in your Calculator component, having to different Components in the same file does not make much sense, please fix it :heavy_check_mark: